### PR TITLE
Update build scripts to use the installed node

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

**Comment by arv** [comment]
PTAL

**Comment by arv** [comment]
Jenkins: Please test this at your convenience... 🙏

**Comment by arv** [comment]
jenkins: test this

**Comment by arv** [comment]
jenkins: test this

File: tools/run_pr_builder.sh [hunk 1]
```diff
@@ -9,7 +9,6 @@
 set -eux
 
 export GOPATH=${WORKSPACE}
-export PATH=${PATH}:/usr/local/go/bin:/var/lib/jenkins/node-v5.11.1-linux-x64/bin
```
Comment (zcstarr) [inline tools/run_pr_builder.sh]: Is it no longer necessary for this script to add node to the path ? Is it handled upstream by the calling process ?  You might have mentioned this in IRL, just want to dbl check.

**Comment by arv** [comment]
Any more comments?

**Pull request merged**